File: C#-part-2/06.Strings-and-Text-Processing/02.Reverse string/ReverseString.cs
//Write a program that reads a string, reverses it and prints the result on the console.
namespace _02.Reverse_string
{
    using System;
    using System.Linq;

    class ReverseString
    {
        static void Main()
        {
            var input = Console.ReadLine().ToArray();
            Array.Reverse(input);
            Console.WriteLine(input);
        }
    }
}

File: JavaScript-Fundamentals/09.Strings/01.Reverse string/Reverse string.js
/**   Reverse string
 * Write a JavaScript function that reverses a string.
 */
function reverse(str) {
    String.prototype.reversed = function reversed() {
        var r = "";
        for (var i = this.length - 1; i >= 0; i--) {
            r += this[i];
        }
        return r;
    };

    console.log(str[0].reversed());
}

reverse(['sample']);